client/src/App.js

```diff
@@ -31,7 +30,7 @@ const App = () => {
         });
       });
     });
-  }, [genres.selectedGenre, spotify.ClientId, spotify.ClientSecret]);
+  }, [genres.selectedGenre, process.env.REACT_APP_SPOTIFY_ID, process.env.REACT_APP_SPOTIFY_SECRET]);
```

Comment:
we shouldn't need to pass the credentials here since they are not going to change

Reply:
good point :) will remove